gauntlet/packages/gauntlet-solana-contracts/src/commands/contracts/ocr2/inspection/inspect.ts

```diff
@@ -224,6 +253,20 @@ export default class OCR2Inspect extends SolanaCommand {
         `Offchain Config "reportingPluginConfig.deltaCNanoseconds"`,
       ),
     ]
+    // Print on-chain oracle information
+    onChainState.oracles.xs.forEach((oracle) => {
+      logger.info(
+        'Oracle ' +
+          oracle.transmitter +
+          '\n- Proposed Payee: ' +
+          oracle.proposedPayee +
+          '\n- From Round ID: ' +
+          oracle.fromRoundId +
+          '\n- Payment Gjuels: ' +
+          oracle.paymentGjuels +
+          '\n',
```

Comment:
JS give us this string type which is more flexible
```suggestion
      logger.info(`Oracle Info:
          - Transmitter: oracle.transmitter
          - Proposed Payee: oracle.proposedPayee
          - From Round ID: oracle.fromRoundId
          - Payment Gjuels: oracle.paymentGjuels
          `)
```